### ISSUE TITLE
Black screen when changing volume with fullscreen content

### ISSUE THREAD

**Issue by megaspel** [ModernFlyouts-Community/ModernFlyouts#683]
Chrome and Internet Explorer edge both suffer this bug when viewing fullscreen content, the screen will go black for a couple seconds every time the volume is nudged, trigging the flyouts menu.

This could be a HDCP issue, since it occurs with Netflix and Disney+

Using a 4K HDR screen with HDCP 2.2 on Windows 10.

**Comment by Cyberdroid1** [comment]
Yeah, this seems to be related to HDCP, since it does not happen with, say, Github.

Does not seem like an issue then :)

**Comment by megaspel** [comment]
How is it not an issue if modernflyouts breaks HDCP content?

**Comment by Cyberdroid1** [comment]
Try opening XBox game bar (and start recording). Does it produce the same black window?


@ShankarBUS what are your views on this?

**Comment by Samuel12321** [comment]
> Does not seem like an issue then :)

How is an obvious bug not an issue?

**Comment by ShankarBUS** [comment]
@megaspel, lmao what?

What is HDCP?

**Comment by megaspel** [comment]
> 
> 
> @megaspel, lmao what?
> 
> What is HDCP?

High-bandwidth Digital Content Protection, it's basically what Netflix/Disney + and the like use to try to stop people from copying their content. It doesn't seem to stop anyone, but it's an added complication that we all have to deal with unfortunately. 

Your monitor and your motherboard has HDCP components, and you need a HDCP compatible cable in order to view netflix or other online TV services, it's gotten to the point where it's common enough that you mostly don't notice it. Nearly all modern TVs, monitors, motherboards and cables, have the right components. Windows default flyout, does not trigger this bug, so it's definitely possible to stay compatible with HDCP.

**Comment by megaspel** [comment]
Turns out this is any video, and I think maybe Chrome specific. If I go fullscreen with firefox with any video, this bug doesnt apply, but with chrome any fullscreen video will turn black when the volume is changed.

**Comment by cartisforearm** [comment]
i have the same issue with all the games i play in fullscreen. i like to have the flyout on the same screen as my game, but it flashes black for a split second twice. it turns black once the flyout pops out and black again once it goes away. i do not think it is HDCP related
